### PR TITLE
Bug 1200142 - Use MozInputMethodManager#setSupportsSwitchingTypes()

### PR DESCRIPTION
**Pull request by timdream**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b1ed0e0bd4fbb87a26c0da2a5383ef9986596854

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=86a645ad49ae553df4886c7f9b4b5fcc54d40c60

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=eb59310e079cd770db180350fca4febf950bc69d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=431342ba5a8d114292ca65647330080109c2d14e

File: apps/system/test/unit/input_layouts_test.js [hunk 1]
```diff
@@ -312,7 +301,7 @@ suite('InputLayouts', function() {
       this.sinon.stub(inputLayouts, '_insertFallbackLayouts');
 
     var stubEmitLayoutsCount =
```
Comment (albertopq) [inline apps/system/test/unit/input_layouts_test.js]: nit: let's change the name of the var as well

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e18a460d7c83789f7099e36d709eadfe800ee0a3

**Pull request merged**